web/shared/TextInput.tsx

```diff
@@ -47,12 +52,25 @@ const TextInput: Component<{
     updateCount()
   })
 
+  createEffect(() => {
+    if (props.isMultiline) {
+      value()
+      resize()
```

Comment:
I need this effect to react to `value` but don't actually need `value`'s value.  This is less explicit than I'd like but I dunno what the idiomatic way to do this in SolidJS is.

Google suggested wrapping this in `on`, as in `on(value, () => resize())` but that did not work for reasons that are unclear to me.